src/PageContent/Calendar/DescriptionContent.js

```diff
@@ -0,0 +1,206 @@
+                );
+            });
+        }
+        return (
+            <div className="wrapper">
+                <img className="content-icon" src={groceries} alt="volunteer" />
+                <div className="content-wrapper content-wrapper-description">
+                    <h1 className="section-header">Donation Description</h1>
+                    {this.state.editedBy ? (
+                        <p className="edited">
+                            {' '}
+                            Edited by {this.state.editedBy},{' '}
+                            {this.state.editedAt}
+                        </p>
+                    ) : null}
+                    {!this.state.edit ? (
+                        <div>
+                            {this.state.donationObject.length > 0 ? (
+                                <div className="content-details-wrapper">
+                                    <p className="content-details description-content">
+                                        {donation}
+                                    </p>
+                                </div>
+                            ) : this.state.accountType ===
+                            AccountType.DONATING_AGENCY_MEMBER ? (
+                                    <button
+                                        className="add-food"
+                                        onClick={this.addRow}
+                                    >
+                                    Add Food Items
+                                    </button>
+                                ) : (
+                                    <p>To Be Determined</p>
+                                )}{' '}
+                        </div>
+                    ) : (
+                        <div className="content-details-wrapper">
+                            <form
+                                className="edit-dg"
+                                onSubmit={this.handleChange}
+                            >
+                                <div className="input-wrapper">
+                                    <div className="food-label">
+                                        Food Item<span className="required">
+                                            *
+                                        </span>
+                                    </div>
+                                    <div className="food-weight-label">
+                                        Weight<span className="required">
+                                            *
+                                        </span>
+                                    </div>
+                                    {editDonation}
+                                </div>
+                                <img
+                                    src={plus}
+                                    alt="plus"
+                                    className="plus"
+                                    onClick={this.addRow}
+                                />
```

Comment:
I think we should also support deleting a row? right now if you add 3 empty rows, we include those (we'll display `... 0lb, 0lb, ....` and when you hit edit again the empty rows are there). I imagine adding a remove row function could be a bit too much work for now, so I think maybe the easiest thing to do now is add a check in `handleChange()` to check whether the donation `name` is empty, if it's empty then don't add it. This way, at least when you hit `save` and then hit `edit`, the empty rows go away and we won't display `0lb` type of info in the text.